crates/matrix-sdk-indexeddb/src/crypto_store/migrations.rs

```diff
@@ -13,15 +13,67 @@
 // limitations under the License.
 
 use indexed_db_futures::{prelude::*, web_sys::DomException};
-use tracing::info;
+use matrix_sdk_crypto::olm::InboundGroupSession;
+use tracing::{debug, info};
 use wasm_bindgen::JsValue;
 
-use crate::crypto_store::{keys, Result};
+use crate::{
+    crypto_store::{
+        indexeddb_serializer::IndexeddbSerializer, keys, InboundGroupSessionIndexedDbObject, Result,
+    },
+    IndexeddbCryptoStoreError,
+};
+
+mod old_keys {
+    /// Old format of the inbound_group_sessions store which lacked indexes or a
+    /// sensible structure
+    pub const INBOUND_GROUP_SESSIONS_OLD: &str = "inbound_group_sessions";
+}
```

Comment:
Instead of introducing another module, could we use (sub)version names in the const names, instead?

```rust
pub const INBOUND_GROUP_SESSIONS_V1: &str = "inbound_group_sessions";
pub const INBOUND_GROUP_SESSIONS_V2: &str = "inbound_group_sessions_v2";
```

Reply:
yes, if you like. The reasons I went with a separate module were twofold:

* Consistency with `state_store` which has [a separate module](https://github.com/matrix-org/matrix-rust-sdk/blob/0344d6a44c8bedd93f361778fa4d34e7dac808a7/crates/matrix-sdk-indexeddb/src/state_store/migrations.rs#L70)
* Reducing the risk of a footgun incident where the "old" store name is accidentally referenced outside the migration code.

Reply:
Ok, that's a fair argument. How about tweaking the names, in case we have other new `inbound_group_sessions` table in the future:

```rust
mod old_keys {
    /// Old format of the inbound_group_sessions store which lacked indexes or a
    /// sensible structure
    pub const INBOUND_GROUP_SESSIONS_V1: &str = "inbound_group_sessions";
}

pub const INBOUND_GROUP_SESSIONS: &str = "inbound_group_sessions_v2";
```

and comments can mention v1 / v2 etc?

Reply:
https://github.com/matrix-org/matrix-rust-sdk/pull/2885/commits/6fd3a69e43801f138dae108c17b1bd6c61c7b611 changed to `_V1` and `_V2` - is that ok?